packages/api/src/config.js

```diff
@@ -298,4 +298,50 @@ export const products = {
     ],
     monthlyPrice: 0,
   },
+  prod_OTTwpzjA4U8B2P: {
+    order: 12,
+    name: "Enterprise",
+    lookupKeys: ["enterprise_plan_1"],
+    usage: [
+      {
+        name: "Transcoding",
+        description: "Transcoding (minutes)",
+        price: 0,
+      },
+      {
+        name: "Delivery",
+        description: "Delivery (minutes)",
+        price: 0,
+      },
+      {
+        name: "Storage",
+        description: "Storage (minutes)",
+        price: 0,
+      },
+    ],
+    monthlyPrice: 0,
+  },
+  prod_OTTbwpzxNLMNSh: {
```

Comment:
Curious what's the difference between the 2?

OK, I saw below that one of them is the staging? Can you add any reference here to specify that while we don't store this in the DB properly? `Enterprise (Staging)` could already do it

Reply:
Right, one is staging, I will change the name to specify it